infrastructure/export.py

```diff
@@ -236,4 +255,43 @@ def refresh_views():
                     AS related
                 ON p.id = related.project_id;
             '''.format(status_cases=status_cases, **case_statements))
+        cursor.execute('''
+            CREATE OR REPLACE VIEW infrastructure_power_plant_export_view AS
+                SELECT pp.id, 
+                pp.name,
+                array_to_string(array_agg(icv.countries::text), ','::text, 'NULL'::text) as countries,
+                array_to_string(array_agg(irv.regions::text), ','::text, 'NULL'::text) as regions,
+                array_to_string(array_agg(lgeo.centroid::text), ','::text, 'NULL'::text) as centroid,
+                array_to_string(array_agg(ippov.owners::text), ','::text, 'NULL'::text) as owners,
+                array_to_string(array_agg(ippopv.operators::text), ','::text, 'NULL'::text) as operators,
+                pp.slug,
+                pp.status,
+                pp.plant_year_online,
+                pp.plant_month_online,
+                pp.plant_day_online,
+                pp.decommissioning_year,
+                pp.decommissioning_month,
+                pp.decommissioning_day,
+                pp.plant_capacity,
+                pp.plant_capacity_unit,
+                pp.plant_output,
+                pp.plant_output_unit,
+                pp.plant_output_year,
+                pp.estimated_plant_output,
+                pp.estimated_plant_output_unit,
+                pp."plant_CO2_emissions",
+                pp."plant_CO2_emissions_unit",
+                pp.grid_connected,
+                pp.description,
+                pp.created_at,
+                pp.updated_at,
+                pp.published
+            FROM infrastructure_powerplant pp
+                LEFT OUTER JOIN infrastructure_countries_view icv ON icv.project_id = pp.id
```

Comment:
Joining where ``icv.project_id = pp.id`` seems wrong. We should be joining on power plant = power plant. We need a new sub view for this, since the existing countries view only concatenates projects, not power plants.